lib/keystores.js

```diff
@@ -169,7 +189,7 @@ api.get = async ({id}) => {
     throw new BedrockError(
       'Keystore configuration not found.',
       'NotFoundError',
-      {dataHub: id, httpStatusCode: 404, public: true});
+      {id, httpStatusCode: 404, public: true});
```

Comment:
So this is a little weird because it's not the `id` of the error or the details, it's the id of the keystore/its config. So maybe `keystore: id`?

```suggestion
      {keystore: id, httpStatusCode: 404, public: true});
```

Reply:
since this is not LD, how about `keystoreId`?

Reply:
Ok, I can live with it. My only hesitation has to do with magic future plans that have never materialized where we convert `BedrockError`s into useful LD. But, that still hasn't happened in many years, so maybe it's not that useful after all.